Cooper/ClientApp/src/app/sign-in/sign-in.component.ts

```diff
@@ -15,7 +15,7 @@ export class SignInComponent {
 
   login(form: NgForm) {
     let credentials = JSON.stringify(form.value);
-    this.http.post("http://localhost:54195/api/auth/login", credentials, {
+    this.http.post("http://localhost:60956/api/auth/login", credentials, {
```

Comment:
Where are these ports from?)

Reply:
Port is automatically generated by Visual Studio in folder - properties, file -  "launchSettings", because in Git this file is ignored